
float64 x
float64 y
float64 theta
float64 d_theta
float64 vel_l
float64 vel_r
uint16 counter_l
uint16 counter_r